src/components/EditorAndOutput.vue

```diff
@@ -47,7 +47,6 @@ import WebGpu from "@/components/WebGpu.vue";
 const props = defineProps<{
   fs: ReactiveFilesystem;
   canvas: HTMLCanvasElement;
-  visual: boolean;
```

Comment:
Instead of "visual" being a global state, it is dependent on which file was opened.